test/expectations/document_highlight/class_and_reference.exp.json

```diff
@@ -19,6 +19,19 @@
             },
             "kind": 3
         },
+        {
```

Comment:
Because of the structure of the tree, we get a duplicate highlight for classes. One occurs because of the `ClassNode` and another because of the `ConstantReadNode` inside of it.

However, the editor dedups this and only shows the write highlight, which results in exactly the same experience for developers.